src/main/java/it/smartcommunitylab/aac/webauthn/auth/WebAuthnAuthenticationException.java

```diff
@@ -0,0 +1,11 @@
+package it.smartcommunitylab.aac.webauthn.auth;
+
+import org.springframework.security.core.AuthenticationException;
+
+public class WebAuthnAuthenticationException extends AuthenticationException {
+
+    public WebAuthnAuthenticationException(String msg) {
+        super(msg);
+    }
+
```

Comment:
align this class with internalauthenticationexception with messages, wrapped exception etc.